devops/test_configs.json

```diff
@@ -1,5 +1,17 @@
 {
   "lts": [
+    {
+      "config": "l0_ocl_gen9",
```

Comment:
I would call it `lin_intel_gen9` or just `lin_gen9`. I think the purpose is to test all existing back-ends at once. It might be more than ocl and l0 back-ends (e.g. host device).